system_baseline/views/admin.py

```diff
@@ -35,21 +35,21 @@ def status():
 
     for i in range(1, 5):
         baseline_bucket_counts[i] = (
-            SystemBaseline.query.with_entities(SystemBaseline.account)
-            .group_by(SystemBaseline.account)
+            SystemBaseline.query.with_entities(SystemBaseline.org_id)
+            .group_by(SystemBaseline.org_id)
```

Comment:
do we need an if/else here or are we sure all will be populated with org_id?

Reply:
This is part of reporting, answered above.